.gitlab-ci.yml

```diff
@@ -150,7 +150,7 @@ before_script:
 # run tests for deb-x64
 run_tests_deb-x64:
   stage: source_test
-  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/deb_x64:latest
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/deb_x64:v1581559-c7ff053
```

Comment:
We should use the images created after the merge to `master` - I'd update these with the new tags _after_ we merge https://github.com/DataDog/datadog-agent-buildimages/pull/12

Reply:
Yes, that was the plan !